src/material/stepper/testing/step-harness-filters.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {BaseHarnessFilters} from '@angular/cdk/testing';
+
+/** Possible orientations for a stepper. */
+export const enum StepperOrientation {HORIZONTAL, VERTICAL}
+
+/** A set of criteria that can be used to filter a list of `MatStepHarness` instances. */
+export interface StepHarnessFilters extends BaseHarnessFilters {
+  /** Only find instances whose label matches the given value. */
+  label?: string | RegExp;
+  /** Only find selected steps. */
```

Comment:
Technically this is something more like: `Only find steps with the given selected state`, since {selected: false} returns all unselected steps, not all steps. Same for the other bools below